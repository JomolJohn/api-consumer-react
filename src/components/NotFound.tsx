const NotFound = () => {
    return (
        <>
        <h4>404- Page not Found</h4>
        </>
    );
};
export default NotFound;