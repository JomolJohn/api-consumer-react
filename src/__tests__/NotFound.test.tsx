import { render } from "enzyme";
import enzymeToJson from "enzyme-to-json";
import NotFound from "../components/NotFound";

describe(" NotFound Component", () => {
    it("should render NotFound component", () => {
        const component = render(<NotFound />);
        expect(enzymeToJson(component)).toMatchSnapshot();
    });
});