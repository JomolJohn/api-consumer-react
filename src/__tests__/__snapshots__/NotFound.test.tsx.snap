// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` NotFound Component should render NotFound component 1`] = `
<h4>
  404- Page not Found
</h4>
`;
